juniper_hyper/src/lib.rs

```diff
@@ -376,8 +401,12 @@ mod tests {
         }
     }
 
-    async fn run_hyper_integration(is_sync: bool) {
-        let port = if is_sync { 3002 } else { 3001 };
+    static mut PORT: u16 = 3001;
+    async fn run_hyper_integration(is_sync: bool, is_custom_type: bool) {
+        let port = unsafe {
```

Comment:
Why this `unsafe`? I don't think this is worth adding more unsafe to the codebase but I could be convinced.

Reply:
Never mind why it's there. I've removed it since it can be avoided.

Please check again @LegNeato